tasks/verify-enterprise-contract-v2.yaml

```diff
@@ -70,15 +70,14 @@ spec:
       image: appstudio-utils
       command: [ec]
       args:
-        - eval
-        - "--input"
+        - validate
+        - image
+        - "--json-input"
         - "$(params.IMAGES)"
         - "--policy"
         - "$(params.POLICY_CONFIGURATION)"
         - "--public-key"
         - "$(params.PUBLIC_KEY)"
-        - "--rekor-url"
```

Comment:
The default value for this parameter in the ec-cli is `https://rekor.sigstore.dev/`. Removing this will force the public instance of rekor to be used, always.

How about we change the default value of REKOR_HOST in this task to an empty string?